tests/alternate_success_code/tests/test.rs

```diff
@@ -0,0 +1,12 @@
+use test_alternate_success_code::*;
+use Windows::Win32::Foundation::CO_E_NOTINITIALIZED;
+use Windows::Win32::System::Com::DoDragDrop;
+
+// This test validates that the AlternateSuccessCodes attribute is being honored by preserving the original signature
+// and not doing any transformation.
+#[test]
+fn test() {
+    let mut effect = 0;
+    let hr = unsafe { DoDragDrop(None, None, 1, &mut effect) };
+    assert!(hr == CO_E_NOTINITIALIZED);
```

Comment:
This is likely to cause the test to spuriously fail. While true in isolation, the condition is destined to fall victim to the [curse of the implicit MTA](https://devblogs.microsoft.com/oldnewthing/20130419-00/?p=4613). With tests getting executed in parallel running out of luck is a mere matter of time.

Reply:
I was under the impression that each test crate is executed in its own process.

Reply:
That's correct, and at this time there is no opportunity for this test to fail. I was rather concerned about future additions to this test crate with the potential to break unrelated test code.

There's no immediate need to change this test, other than—maybe—leaving a comment stating the implicit assumptions about the test conditional.

Reply:
The tests are certainly a bit brittle. That's why I am in the process of writing a standalone test component in C++, much like I have for cppwinrt. That should address this concern.